packages/components/src/components/buttons/ListItemAction.tsx

```diff
@@ -64,7 +64,7 @@ const ArrowExpansionSecion = styled(ArrowExpansionButton)<{
   ${({ isFullHeight }) => isFullHeight && ` height: 100%;`}
 `
 const ActionButtonContainer = styled.div`
-  width: 36px;
+  width: 40px;
```

Comment:
Increase width keeping same with height, otherwise, the circle button gets an uneven radius having the shape of an ellipse